### PR TITLE
updates: stable and testing streams rollout on 2020-10-20

### PR DESCRIPTION
**Pull request by sinnykumari**
stable: 32.20201004.3.0
testing: 32.20201018.2.0

### CODE REVIEW

File: streams/stable.json [hunk 1]
```diff
@@ -185,92 +185,16 @@
             "images": {
                 "aws": {
                     "regions": {
```
Comment (dustymabe) [inline streams/stable.json:187]: looks like we lost aws replication somehow

Comment (sinnykumari) [reply streams/stable.json:187]: good catch Dusty!

Comment (sinnykumari) [reply streams/stable.json]: should be fixed now.

**Comment by dustymabe** [comment]
```
stable
    version: 32.20201004.3.0
    start: 2020-10-20 21:00:00+00:00 UTC (in 1:41:39)
           2020-10-20 17:00:00-04:00 Raleigh/New York/Toronto
           2020-10-20 23:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
testing
    version: 32.20201018.2.0
    start: 2020-10-20 21:00:00+00:00 UTC (in 1:41:39)
           2020-10-20 17:00:00-04:00 Raleigh/New York/Toronto
           2020-10-20 23:00:00+02:00 Berlin/France/Poland
    duration: 2880m (48.0h)
```

LGTM

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by jlebon — APPROVED** [review]

**Pull request merged**